docs/install.md

```diff
@@ -434,14 +434,16 @@ Select the following tabs for the installation description of your target.
         Click **Next**.
 
     1. Tab **System**: As **Machine** you can select `q35`, but the older default will work as well. We recommend the para-virtualised **VirtIO SCSI** controller, which should be the default.  
-        Activate the check box `Qemu Agent`.  
+        _**Important:** Activate the check box `Qemu Agent`._  
         Click **Next**. 
     
     1. Tab **Disks**: Optionally, change the VM disk storage location ("Storage"), optionally adjust the disk size ("Disk size (GiB)").  
+        _**Important:** A disk size of at least 8 GB is required._  
```

Comment:
It should work as well with 4 GiB or less, if 2 GiB RAM or more are available (hence no swap file created), or is there another problem with smaller disk sizes?

Reply:
The problem arises during the Clonezilla operation. Some of the partition write operations did stop with failures.

Reply:
Ah okay, good to know.